.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,95 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+    branches:
+      - "*.x"
+      - "master"
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit:
+    name: "PHPUnit"
+    runs-on: "ubuntu-20.04"
+
+    services:
+      mongodb:
+        image: mongo
+        ports:
+          - 27017:27017
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        dependencies:
+          - "highest"
+        include:
+          - php-version: "7.2"
+            dependencies: "lowest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP with Xdebug"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          extensions: "mongodb, sqlite3"
+          coverage: "xdebug"
+          ini-values: "zend.assertions=1"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v2"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        uses: "ramsey/composer-install@v1"
+        with:
+          dependency-versions: "${{ matrix.dependencies }}"
```

Comment:
I found this more readable than manually composing Composer commands with and without `--prefer-lowest`, [as is done in doctrine/orm](https://github.com/doctrine/orm/blob/242cf1a33df1b8bc5e1b86c3ebd01db07851c833/.github/workflows/continuous-integration.yml#L49-L55) for instance. What do you think?

Reply:
I think it looks great, didn't know about that action!
The README says

> It installs your Composer dependencies and caches them for improved build times.

Does that mean the cache step should be removed?

Reply:
> There is no need to set up a separate caching step since ramsey/composer-install handles this for you.

It clearly needs to be removed.

Reply:
Please open a PR against https://github.com/doctrine/.github/ so that we can discuss this at the org-wide level. Please link add a link to this PR to demonstrate how much it improves the situation.

Reply:
Nice, I missed that!

If you like, I can use this action in all workflows in this and subsequent PRs.

Reply:
> Please open a PR against https://github.com/doctrine/.github/ so that we can discuss this at the org-wide level. Please link add a link to this PR to demonstrate how much it improves the situation.

Done: https://github.com/doctrine/.github/pull/16

Reply:
> If you like, I can use this action in all workflows in this and subsequent PRs.

I would like that, but let's see if the others would like that too.

Reply:
Resolving as https://github.com/doctrine/.github/pull/16 has been approved.